engine/src/main/java/org/terasology/logic/ai/SimpleAISystem.java

```diff
@@ -88,7 +89,7 @@ public void update(float delta) {
                 drive.set(targetDirection);
 
                 float yaw = (float) Math.atan2(targetDirection.x, targetDirection.z);
-                location.getLocalRotation().set(0, 1, 0, yaw);
+                location.setLocalRotation(new Quaternionf().setAngleAxis(yaw, 0, 1, 0));
```

Comment:
I changed this. I'm pretty sure this is suppose to be axis angle.

Reply:
After seeing this bug in GooKeeper I'm pretty sure, too :sweat_smile: